.github/workflows/unit-test.yml

```diff
@@ -0,0 +1,28 @@
+name: Unit-tests
+
+on:
```

Comment:
Should the unit-tests (which I assume run rather quickly) also run on the default/master branch? And not only on PRs?